### PR TITLE
Feature/#2 create login page

### PR DESCRIPTION
**Pull request by deliacpopescu**
I created Login Page. 
Please review.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -14,4 +14,11 @@
     <div id="app"></div>
     <!-- built files will be auto injected -->
   </body>
+  <style>
+    body{
```
Comment (mariangoia91) [inline public/index.html:18]: please add space between bracket and identifier

File: src/App.vue [hunk 2]
```diff
@@ -1,20 +1,25 @@
...
+  <Login></Login>
+  <Main></Main>
   <SelectCV></SelectCV>
 </template>
 
 <script>
 import Login from "./components/Login.vue";
 import Main from "./components/Main.vue";
 import SelectCV from "./components/SelectCV.vue";
+import "bootstrap/dist/css/bootstrap.css";
```
Comment (mariangoia91) [inline src/App.vue:11]: please import bootstrap using npm

File: src/components/Login.vue [hunk 3]
```diff
@@ -0,0 +1,81 @@
+<template>
+  <div class="wrapper">
+    <div class="fields-wrapper">
+      <div class="in align-items-center">
+        <label class="p-3" for="username"><strong>Username</strong></label>
```
Comment (mariangoia91) [inline src/components/Login.vue:5]: please separate html tags with newlines

File: src/components/Login.vue [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+.submit-btn {
+  position: relative;
+  width: 6rem;
+  height: 10%;
+  /* left: 40%; */
+  background: #00adb5;
+  border-radius: 50px;
+  margin-top: 2rem;
+}
+.in {
```
Comment (mariangoia91) [inline src/components/Login.vue:69]: please use a more descriptive name

File: src/components/Login.vue [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+</script>
+
+<style>
+.wrapper {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  background: #393e46;
+}
+.fields-wrapper {
```
Comment (mariangoia91) [inline src/components/Login.vue:35]: missing newline

File: src/components/Login.vue [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+          placeholder="Password"
+        />
+      </div>
+      <button class="submit-btn">Login</button>
+    </div>
+  </div>
+</template>
+
+<script>
+import "bootstrap/dist/css/bootstrap.css";
```
Comment (mariangoia91) [inline src/components/Login.vue:23]: use npm to import bootstrap

**Review by tant-rares-30127 — APPROVED** [review]
Nice job! Take care of commented code. Signed, Tant, Claudia, Teona.

File: src/components/Login.vue [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+.fields-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: relative;
+  width: 30%;
+  height: 40%;
+  top: 30%;
+  left: 35%;
+  background: #eeeeee;
```
Comment (mariangoia91) [inline src/components/Login.vue:44]: please store colors as constants so they have a descriptive name

File: src/main.js [hunk 8]
```diff
@@ -1,18 +1,15 @@
 import { createApp } from 'vue'
 import App from './App.vue'
 
-import router from "./router";
+//import router from "./router";
```
Comment (mariangoia91) [inline src/main.js:4]: remove uncommented code

File: src/components/Login.vue [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+  border-radius: 10px;
+  outline: none;
+  color: black;
+}
+
+.submit-btn {
+  position: relative;
+  width: 6rem;
+  height: 10%;
+  /* left: 40%; */
```
Comment (mariangoia91) [inline src/components/Login.vue:64]: remove uncommented code

**Review by mariangoia91 — CHANGES_REQUESTED** [review]

**Pull request merged**